allure-commandline/src/main/scripts/allure

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [ -z "$ALLURE_HOME" ]; then
+
+    SOURCE="${BASH_SOURCE[0]}"
+    while [ -h "$SOURCE" ]; do
+        DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+        SOURCE="$(readlink "$SOURCE")"
+        [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE"
+    done
+    DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+    ALLURE_HOME="$DIR/.."
+
+fi
+
+ALLURE_COMMANDLINE_JAR="$ALLURE_HOME/lib/allure-commandline.jar"
+
+if [ ! -f "$ALLURE_COMMANDLINE_JAR" ]; then
+
+    echo "$ALLURE_HOME: is not a valid Allure home directory"
+    exit 1
+
+fi
+
+convert_env_variable_to_java () {
+    echo "-D$2=$1"
+}
+
+if [ ! -z "$ALLURE_REPORT_JAVA_OPTS" ]; then
+    ALLURE_COMMANDLINE_ENV=$(convert_env_variable_to_java ${ALLURE_REPORT_JAVA_OPTS} "allure.report.javaOpts")
```

Comment:
think you can just parse env vars without converting them

https://github.com/yandex-qatools/juseppe/tree/master/src/main/java/ru/lanwen/jenkins/juseppe/props